build_stats.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+set -e
+
+HERE_DIR="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
+INST_DIR=${HERE_DIR}/R-inst
```

Comment:
Why don't we dump this stuff into `work` so that it doesn't clutter up the top-level of the repository? So I guess it would be something like `INST_DIR=${HERE_DIR}/work/R-inst` and so on.